Entity/Task.php

```diff
@@ -215,14 +215,22 @@ public function setTaskId($taskId)
      */
     public function getCreatorFullName()
     {
-        return $this->getCreator()->getFullName();
+        if (!($creator = $this->getCreator())) {
```

Comment:
Why not simply `!$this->creator`? Or if you don't want to use the protected member variable of the trait something like this:
```php
$creator = $this->getCreator();
if (!$creator) {
    return '';
}

return $creator->getFullName()
```

Negating and assigning in a single if statement does not look really nice to me...